fbfmaproom/assets/style.css

```diff
@@ -61,3 +44,93 @@ tr {
 #command_panel label {
     margin-bottom: 0.5rem;
 }
+
+table.supertable {
+    border-collapse: separate;
+    border-spacing: 0;
+    width: 100%;
+    table-layout: fixed;
+    overflow-y: auto;
+    overflow-x: hidden;
+
+    border-style: solid;
+    border-width: 1px;
+    border-color: rgb(200, 200, 200);
+
+    text-align: center;
+    vertical-align: center;
+    font-size: 14px;
+    font-family: monospace;
+}
+
+table.supertable td, table.supertable th {
+    border-right-style: solid;
+    border-right-width: 1px;
+    border-right-color: rgb(200, 200, 200);
+
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    border-bottom-color: rgb(200, 200, 200);
+
+    padding: 6px;
+}
+
+table.supertable th {
+    background-color: white;
+    font-weight: bold;
+    color: rgb(251, 101, 57);
+}
+
+table.supertable tr th:first-child {
+    color: rgb(67, 104, 176);
+}
+
+table.supertable thead tr:last-child th {
+    background-color: rgb(241, 241, 241);
+    color: black;
+}
+
+table.supertable td {
+    background-color: rgb(248, 248, 248);
+}
+
+table.supertable tr:last-child td {
+    border-bottom: 0px;
+}
+
+.cell-el-nino {
+    background-color: rgb(172, 23, 25) !important;
```

Comment:
These classes are dynamically added and the usually rules for precedence don't see to apply in such circumstances. This was the only way I could really get it to work. It isn't *quite* a hack inasmuch as its there for a consistent reason.

Reply:
Not a priority right now, but at some point I would like to have a better understanding of the precedence rules than "this seems to work."

According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity):

> Using !important, however, is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets.

Reply:
Figured out a better way that removes !important